### PR TITLE
py-slvs: Init at 1.0.1

### PR DESCRIPTION
**Pull request by jvanbruegge**
Sadly, there are no clean urls for this package available, so
fetchFromPypi does not work

###### Motivation for this change
This binding is needed for the [assembly3 workbench](https://github.com/realthunder/FreeCAD_assembly3) of freecad

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -5518,6 +5518,16 @@
     githubId = 2396926;
     name = "Justin Woo";
   };
+  jvanbruegge = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:5521]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

File: pkgs/development/python-modules/py-slvs/default.nix [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      py27 = {
+        url = "https://files.pythonhosted.org/packages/4a/0d/47e630a8f3c01956aedf083705b11d709fb943e8471607d0dcd56c0642f5/py_slvs-1.0.1-cp27-cp27m-manylinux1_x86_64.whl";
+        sha256 = "c2ec1491825a8b005d8ec8d276b9069aa8a7d82f0ea41e819e15e49e44a8bf51";
+      };
+    };
+  };
+
+key = with python.passthru; if isPy39 then
+  "py39" else if isPy38 then "py38" else if isPy37 then "py37" else if isPy36 then "py36"
+  else if isPy35 then "py35" else "py27";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/py-slvs/default.nix:85]: Why not use python.version?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Why are we not building from source?

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (jvanbruegge) [reply maintainers/maintainer-list.nix:5521]: done

File: pkgs/development/python-modules/py-slvs/default.nix (hunk 2 above)
Comment (jvanbruegge) [reply pkgs/development/python-modules/py-slvs/default.nix]: Was not aware of that. Done

**Comment by jvanbruegge** [comment]
I am not building from source, because I have no idea how to build the native component 😅

**Comment by jonringer** [comment]
> I am not building from source, because I have no idea how to build the native component sweat_smile

you will mostly like need to expose these dependencies https://github.com/realthunder/slvs_py/blob/master/pyobject.toml, and pull from source

**Comment by jvanbruegge** [comment]
I will close this as I currently do not have the time to figure out how to do this
